.github/workflows/main.yml

```diff
@@ -16,3 +16,10 @@ jobs:
       linux_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
       linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
       linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+
+  plugin-tests:
+    name: Plugin tests
+    uses: apple/swift-nio/.github/workflows/swift_load_test_matrix.yml@main
```

Comment:
On first read I thought "why are we doing load testing!?" 🙃

Reply:
Ah right, no it's "load test matrix", not "load test matrix".